server.js

```diff
@@ -73,6 +107,15 @@ const WALKTHROUGH_LOCATION_DEFAULT = {
 const walkthroughs = [];
 let server;
 
+app.get('/services', (req, res) => {
+  res.setHeader('Content-Type', 'application/json');
+  if (process.env.INSTALLED_SERVICES) {
```

Comment:
verify this doesn't break OS3/OS4 live system

Reply:
Yep.